docs/sql-ref-syntax-ddl-repair-table.md

```diff
@@ -26,7 +26,7 @@ license: |
 ### Syntax
 
 ```sql
-MSCK REPAIR TABLE table_identifier
+MSCK REPAIR TABLE table_identifier [[ADD|DROP|SYNC] PARTITIONS]
```

Comment:
This looks inconsistent from the implemented one. Do you want `[[ADD|DROP|SYNC] PARTITIONS]` -> `[(ADD|DROP|SYNC) PARTITIONS]` because we always need one of `ADD/DROP/SYNC`?

Reply:
Well, could you check the PR title together?

Reply:
Will change it to `[{ADD|DROP|SYNC} PARTITIONS]`